docs/source/telegram.chatshared.rst

```diff
@@ -0,0 +1,6 @@
+telegram.ChatShared
```

Comment:
```suggestion
ChatShared
```
we dropped the telegram. prefix in the sidebar so

Reply:
all other files have it